Refresh.GameServer/Database/GameDatabaseProvider.cs

```diff
@@ -168,6 +168,9 @@ protected override void Migrate(Migration migration, ulong oldVersion)
 
             // Version was bumped here to delete invalid favourite level relations
             if (oldVersion < 101) migration.NewRealm.RemoveRange(newUser.FavouriteLevelRelations.Where(r => r.Level == null));
+
+            // In version 102 we split the Vita icon hash from the PS3 icon hash
+            if (oldVersion < 102) newUser.VitaIconHash = oldUser.IconHash;
```

Comment:
Since these systems are incompatible we should reset to `0` here; it's not too big of a deal if we just ask people to reset their avatars on Vita.